src/scss/settings/_variables.scss

```diff
@@ -118,9 +118,9 @@ $zIndex: (
 
 // Path vars
 // ==========================================================================
-$img-path                  : '/assets/dist/img'; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
-$icon-path                 : '#{$img-path}/icons';
-$font-path                 : '/assets/dist/fonts';
+$img-path                  : 'fozzie/dist/img' !default; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
```

Comment:
Although, if we are happy to have just the `sprite.svg` rather than individual assets and classes for every icon that would simplify the matter a lot.